index.html

```diff
@@ -81,7 +101,7 @@ <h6>2015</h6>
             <li><a href="#">JavaScript</a></li>
           </ul>
           <div class="action">
-            <button type="submit" class="btn"><p>See project</p></button>
+            <button type="submit" class="btn" onclick="toggle(0)""><p>See project</p></button>
```

Comment:
Your idea for displaying the cards dynamically is pretty impressive. 👏🏿 . However, after `popup` closes, other `See Project` buttons do not work.

- [ ] Kindly check and correct this error by making sure that the page and its elements return to their original after all popup changes have been removed. 🤏🏿